// @porsche-design-system/components-js/styles/esm/grid/gridStyle.js
import { getMediaQueryMin } from '../mediaQuery/getMediaQueryMin.js';
import { gridGap } from './gridGap.js';
import { gridFullColumnStart } from './gridFullColumnStart.js';
import { gridWideColumnStart } from './gridWideColumnStart.js';
import { gridExtendedColumnStart } from './gridExtendedColumnStart.js';
import { gridBasicColumnStart } from './gridBasicColumnStart.js';
import { gridNarrowColumnStart } from './gridNarrowColumnStart.js';
import { gridNarrowColumnEnd } from './gridNarrowColumnEnd.js';
import { gridBasicColumnEnd } from './gridBasicColumnEnd.js';
import { gridExtendedColumnEnd } from './gridExtendedColumnEnd.js';
import { gridWideColumnEnd } from './gridWideColumnEnd.js';
import { gridFullColumnEnd } from './gridFullColumnEnd.js';
import { _cssVariableGridSafeZone, _gridSafeZoneBase, _cssVariableGridExtendedSpanOneHalf, _cssVariableGridBasicSpanOneHalf, _cssVariableGridBasicSpanOneThird, _cssVariableGridBasicSpanTwoThirds, _cssVariableGridNarrowSpanOneHalf, _gridWidthMin, _gridWidthMax, _gridSafeZoneS, _gridSafeZoneXXL } from './gridShared.js';

const outerColumn = `minmax(0, calc(var(${_cssVariableGridSafeZone}) - ${gridGap}))`;
const column = 'minmax(0, 1fr)';
const getColumns = (repeat) => (repeat > 1 ? `repeat(${repeat}, ${column})` : column);
const getColumnSpan = (span) => `span ${span}`;
const getGridTemplateColumns = (layout) => layout === 'mobile'
    ? `[${gridFullColumnStart}] ${outerColumn} [${gridWideColumnStart} ${gridExtendedColumnStart} ${gridBasicColumnStart} ${gridNarrowColumnStart}] ${getColumns(6)} [${gridNarrowColumnEnd} ${gridBasicColumnEnd} ${gridExtendedColumnEnd} ${gridWideColumnEnd}] ${outerColumn} [${gridFullColumnEnd}]`
    : `[${gridFullColumnStart}] ${outerColumn} [${gridWideColumnStart}] ${getColumns(1)} [${gridExtendedColumnStart}] ${getColumns(1)} [${gridBasicColumnStart}] ${getColumns(2)} [${gridNarrowColumnStart}] ${getColumns(8)} [${gridNarrowColumnEnd}] ${getColumns(2)} [${gridBasicColumnEnd}] ${getColumns(1)} [${gridExtendedColumnEnd}] ${getColumns(1)} [${gridWideColumnEnd}] ${outerColumn} [${gridFullColumnEnd}]`;
const gridStyle = {
    [_cssVariableGridSafeZone]: _gridSafeZoneBase,
    [_cssVariableGridExtendedSpanOneHalf]: getColumnSpan(3),
    [_cssVariableGridBasicSpanOneHalf]: getColumnSpan(3),
    [_cssVariableGridBasicSpanOneThird]: getColumnSpan(2),
    [_cssVariableGridBasicSpanTwoThirds]: getColumnSpan(4),
    [_cssVariableGridNarrowSpanOneHalf]: getColumnSpan(3),
    display: 'grid',
    gridGap,
    gridTemplateColumns: getGridTemplateColumns('mobile'),
    minWidth: _gridWidthMin,
    maxWidth: _gridWidthMax,
    margin: 0,
    padding: `0 calc(50vw - ${_gridWidthMax} / 2)`,
    boxSizing: 'content-box',
    [getMediaQueryMin('s')]: {
        [_cssVariableGridSafeZone]: _gridSafeZoneS,
        [_cssVariableGridExtendedSpanOneHalf]: getColumnSpan(7),
        [_cssVariableGridBasicSpanOneHalf]: getColumnSpan(6),
        [_cssVariableGridBasicSpanOneThird]: getColumnSpan(4),
        [_cssVariableGridBasicSpanTwoThirds]: getColumnSpan(8),
        [_cssVariableGridNarrowSpanOneHalf]: getColumnSpan(4),
        gridTemplateColumns: getGridTemplateColumns('desktop'),
    },
    [getMediaQueryMin('xxl')]: {
        [_cssVariableGridSafeZone]: _gridSafeZoneXXL,
    },
};

export { gridStyle };
