// @porsche-design-system/components-js/styles/grid/gridStyle.js
'use strict';

var getMediaQueryMin = require('../mediaQuery/getMediaQueryMin.js');
var gridGap = require('./gridGap.js');
var gridFullColumnStart = require('./gridFullColumnStart.js');
var gridWideColumnStart = require('./gridWideColumnStart.js');
var gridExtendedColumnStart = require('./gridExtendedColumnStart.js');
var gridBasicColumnStart = require('./gridBasicColumnStart.js');
var gridNarrowColumnStart = require('./gridNarrowColumnStart.js');
var gridNarrowColumnEnd = require('./gridNarrowColumnEnd.js');
var gridBasicColumnEnd = require('./gridBasicColumnEnd.js');
var gridExtendedColumnEnd = require('./gridExtendedColumnEnd.js');
var gridWideColumnEnd = require('./gridWideColumnEnd.js');
var gridFullColumnEnd = require('./gridFullColumnEnd.js');
var gridShared = require('./gridShared.js');

const outerColumn = `minmax(0, calc(var(${gridShared._cssVariableGridSafeZone}) - ${gridGap.gridGap}))`;
const column = 'minmax(0, 1fr)';
const getColumns = (repeat) => (repeat > 1 ? `repeat(${repeat}, ${column})` : column);
const getColumnSpan = (span) => `span ${span}`;
const getGridTemplateColumns = (layout) => layout === 'mobile'
    ? `[${gridFullColumnStart.gridFullColumnStart}] ${outerColumn} [${gridWideColumnStart.gridWideColumnStart} ${gridExtendedColumnStart.gridExtendedColumnStart} ${gridBasicColumnStart.gridBasicColumnStart} ${gridNarrowColumnStart.gridNarrowColumnStart}] ${getColumns(6)} [${gridNarrowColumnEnd.gridNarrowColumnEnd} ${gridBasicColumnEnd.gridBasicColumnEnd} ${gridExtendedColumnEnd.gridExtendedColumnEnd} ${gridWideColumnEnd.gridWideColumnEnd}] ${outerColumn} [${gridFullColumnEnd.gridFullColumnEnd}]`
    : `[${gridFullColumnStart.gridFullColumnStart}] ${outerColumn} [${gridWideColumnStart.gridWideColumnStart}] ${getColumns(1)} [${gridExtendedColumnStart.gridExtendedColumnStart}] ${getColumns(1)} [${gridBasicColumnStart.gridBasicColumnStart}] ${getColumns(2)} [${gridNarrowColumnStart.gridNarrowColumnStart}] ${getColumns(8)} [${gridNarrowColumnEnd.gridNarrowColumnEnd}] ${getColumns(2)} [${gridBasicColumnEnd.gridBasicColumnEnd}] ${getColumns(1)} [${gridExtendedColumnEnd.gridExtendedColumnEnd}] ${getColumns(1)} [${gridWideColumnEnd.gridWideColumnEnd}] ${outerColumn} [${gridFullColumnEnd.gridFullColumnEnd}]`;
const gridStyle = {
    [gridShared._cssVariableGridSafeZone]: gridShared._gridSafeZoneBase,
    [gridShared._cssVariableGridExtendedSpanOneHalf]: getColumnSpan(3),
    [gridShared._cssVariableGridBasicSpanOneHalf]: getColumnSpan(3),
    [gridShared._cssVariableGridBasicSpanOneThird]: getColumnSpan(2),
    [gridShared._cssVariableGridBasicSpanTwoThirds]: getColumnSpan(4),
    [gridShared._cssVariableGridNarrowSpanOneHalf]: getColumnSpan(3),
    display: 'grid',
    gridGap: gridGap.gridGap,
    gridTemplateColumns: getGridTemplateColumns('mobile'),
    minWidth: gridShared._gridWidthMin,
    maxWidth: gridShared._gridWidthMax,
    margin: 0,
    padding: `0 calc(50vw - ${gridShared._gridWidthMax} / 2)`,
    boxSizing: 'content-box',
    [getMediaQueryMin.getMediaQueryMin('s')]: {
        [gridShared._cssVariableGridSafeZone]: gridShared._gridSafeZoneS,
        [gridShared._cssVariableGridExtendedSpanOneHalf]: getColumnSpan(7),
        [gridShared._cssVariableGridBasicSpanOneHalf]: getColumnSpan(6),
        [gridShared._cssVariableGridBasicSpanOneThird]: getColumnSpan(4),
        [gridShared._cssVariableGridBasicSpanTwoThirds]: getColumnSpan(8),
        [gridShared._cssVariableGridNarrowSpanOneHalf]: getColumnSpan(4),
        gridTemplateColumns: getGridTemplateColumns('desktop'),
    },
    [getMediaQueryMin.getMediaQueryMin('xxl')]: {
        [gridShared._cssVariableGridSafeZone]: gridShared._gridSafeZoneXXL,
    },
};

exports.gridStyle = gridStyle;
