// @porsche-design-system/components-js/styles/theme/themeLight.js
'use strict';

var themeLightPrimary = require('./themeLightPrimary.js');
var themeLightBackgroundBase = require('./themeLightBackgroundBase.js');
var themeLightBackgroundSurface = require('./themeLightBackgroundSurface.js');
var themeLightBackgroundShading = require('./themeLightBackgroundShading.js');
var themeLightContrastLow = require('./themeLightContrastLow.js');
var themeLightContrastMedium = require('./themeLightContrastMedium.js');
var themeLightContrastHigh = require('./themeLightContrastHigh.js');
var themeLightNotificationSuccess = require('./themeLightNotificationSuccess.js');
var themeLightNotificationSuccessSoft = require('./themeLightNotificationSuccessSoft.js');
var themeLightNotificationWarning = require('./themeLightNotificationWarning.js');
var themeLightNotificationWarningSoft = require('./themeLightNotificationWarningSoft.js');
var themeLightNotificationError = require('./themeLightNotificationError.js');
var themeLightNotificationErrorSoft = require('./themeLightNotificationErrorSoft.js');
var themeLightNotificationInfo = require('./themeLightNotificationInfo.js');
var themeLightNotificationInfoSoft = require('./themeLightNotificationInfoSoft.js');
var themeLightStateHover = require('./themeLightStateHover.js');
var themeLightStateActive = require('./themeLightStateActive.js');
var themeLightStateFocus = require('./themeLightStateFocus.js');
var themeLightStateDisabled = require('./themeLightStateDisabled.js');

const themeLight = {
    primary: themeLightPrimary.themeLightPrimary,
    background: {
        base: themeLightBackgroundBase.themeLightBackgroundBase,
        surface: themeLightBackgroundSurface.themeLightBackgroundSurface,
        shading: themeLightBackgroundShading.themeLightBackgroundShading,
    },
    contrast: {
        low: themeLightContrastLow.themeLightContrastLow,
        medium: themeLightContrastMedium.themeLightContrastMedium,
        high: themeLightContrastHigh.themeLightContrastHigh,
    },
    notification: {
        success: themeLightNotificationSuccess.themeLightNotificationSuccess,
        successSoft: themeLightNotificationSuccessSoft.themeLightNotificationSuccessSoft,
        warning: themeLightNotificationWarning.themeLightNotificationWarning,
        warningSoft: themeLightNotificationWarningSoft.themeLightNotificationWarningSoft,
        error: themeLightNotificationError.themeLightNotificationError,
        errorSoft: themeLightNotificationErrorSoft.themeLightNotificationErrorSoft,
        info: themeLightNotificationInfo.themeLightNotificationInfo,
        infoSoft: themeLightNotificationInfoSoft.themeLightNotificationInfoSoft,
    },
    state: {
        hover: themeLightStateHover.themeLightStateHover,
        active: themeLightStateActive.themeLightStateActive,
        focus: themeLightStateFocus.themeLightStateFocus,
        disabled: themeLightStateDisabled.themeLightStateDisabled,
    },
};

exports.themeLight = themeLight;
