// @porsche-design-system/components-js/styles/theme/themeDark.js
'use strict';

var themeDarkPrimary = require('./themeDarkPrimary.js');
var themeDarkBackgroundBase = require('./themeDarkBackgroundBase.js');
var themeDarkBackgroundSurface = require('./themeDarkBackgroundSurface.js');
var themeDarkBackgroundShading = require('./themeDarkBackgroundShading.js');
var themeDarkContrastLow = require('./themeDarkContrastLow.js');
var themeDarkContrastMedium = require('./themeDarkContrastMedium.js');
var themeDarkContrastHigh = require('./themeDarkContrastHigh.js');
var themeDarkNotificationSuccess = require('./themeDarkNotificationSuccess.js');
var themeDarkNotificationSuccessSoft = require('./themeDarkNotificationSuccessSoft.js');
var themeDarkNotificationWarning = require('./themeDarkNotificationWarning.js');
var themeDarkNotificationWarningSoft = require('./themeDarkNotificationWarningSoft.js');
var themeDarkNotificationError = require('./themeDarkNotificationError.js');
var themeDarkNotificationErrorSoft = require('./themeDarkNotificationErrorSoft.js');
var themeDarkNotificationInfo = require('./themeDarkNotificationInfo.js');
var themeDarkNotificationInfoSoft = require('./themeDarkNotificationInfoSoft.js');
var themeDarkStateHover = require('./themeDarkStateHover.js');
var themeDarkStateActive = require('./themeDarkStateActive.js');
var themeDarkStateFocus = require('./themeDarkStateFocus.js');
var themeDarkStateDisabled = require('./themeDarkStateDisabled.js');

const themeDark = {
    primary: themeDarkPrimary.themeDarkPrimary,
    background: {
        base: themeDarkBackgroundBase.themeDarkBackgroundBase,
        surface: themeDarkBackgroundSurface.themeDarkBackgroundSurface,
        shading: themeDarkBackgroundShading.themeDarkBackgroundShading,
    },
    contrast: {
        low: themeDarkContrastLow.themeDarkContrastLow,
        medium: themeDarkContrastMedium.themeDarkContrastMedium,
        high: themeDarkContrastHigh.themeDarkContrastHigh,
    },
    notification: {
        success: themeDarkNotificationSuccess.themeDarkNotificationSuccess,
        successSoft: themeDarkNotificationSuccessSoft.themeDarkNotificationSuccessSoft,
        warning: themeDarkNotificationWarning.themeDarkNotificationWarning,
        warningSoft: themeDarkNotificationWarningSoft.themeDarkNotificationWarningSoft,
        error: themeDarkNotificationError.themeDarkNotificationError,
        errorSoft: themeDarkNotificationErrorSoft.themeDarkNotificationErrorSoft,
        info: themeDarkNotificationInfo.themeDarkNotificationInfo,
        infoSoft: themeDarkNotificationInfoSoft.themeDarkNotificationInfoSoft,
    },
    state: {
        hover: themeDarkStateHover.themeDarkStateHover,
        active: themeDarkStateActive.themeDarkStateActive,
        focus: themeDarkStateFocus.themeDarkStateFocus,
        disabled: themeDarkStateDisabled.themeDarkStateDisabled,
    },
};

exports.themeDark = themeDark;
