// @porsche-design-system/components-js/styles/esm/theme/themeDark.js
import { themeDarkPrimary } from './themeDarkPrimary.js';
import { themeDarkBackgroundBase } from './themeDarkBackgroundBase.js';
import { themeDarkBackgroundSurface } from './themeDarkBackgroundSurface.js';
import { themeDarkBackgroundShading } from './themeDarkBackgroundShading.js';
import { themeDarkContrastLow } from './themeDarkContrastLow.js';
import { themeDarkContrastMedium } from './themeDarkContrastMedium.js';
import { themeDarkContrastHigh } from './themeDarkContrastHigh.js';
import { themeDarkNotificationSuccess } from './themeDarkNotificationSuccess.js';
import { themeDarkNotificationSuccessSoft } from './themeDarkNotificationSuccessSoft.js';
import { themeDarkNotificationWarning } from './themeDarkNotificationWarning.js';
import { themeDarkNotificationWarningSoft } from './themeDarkNotificationWarningSoft.js';
import { themeDarkNotificationError } from './themeDarkNotificationError.js';
import { themeDarkNotificationErrorSoft } from './themeDarkNotificationErrorSoft.js';
import { themeDarkNotificationInfo } from './themeDarkNotificationInfo.js';
import { themeDarkNotificationInfoSoft } from './themeDarkNotificationInfoSoft.js';
import { themeDarkStateHover } from './themeDarkStateHover.js';
import { themeDarkStateActive } from './themeDarkStateActive.js';
import { themeDarkStateFocus } from './themeDarkStateFocus.js';
import { themeDarkStateDisabled } from './themeDarkStateDisabled.js';

const themeDark = {
    primary: themeDarkPrimary,
    background: {
        base: themeDarkBackgroundBase,
        surface: themeDarkBackgroundSurface,
        shading: themeDarkBackgroundShading,
    },
    contrast: {
        low: themeDarkContrastLow,
        medium: themeDarkContrastMedium,
        high: themeDarkContrastHigh,
    },
    notification: {
        success: themeDarkNotificationSuccess,
        successSoft: themeDarkNotificationSuccessSoft,
        warning: themeDarkNotificationWarning,
        warningSoft: themeDarkNotificationWarningSoft,
        error: themeDarkNotificationError,
        errorSoft: themeDarkNotificationErrorSoft,
        info: themeDarkNotificationInfo,
        infoSoft: themeDarkNotificationInfoSoft,
    },
    state: {
        hover: themeDarkStateHover,
        active: themeDarkStateActive,
        focus: themeDarkStateFocus,
        disabled: themeDarkStateDisabled,
    },
};

export { themeDark };
